File: 2942-find-words-containing-character/2942-find-words-containing-character.js
/**
 * @param {string[]} words
 * @param {character} x
 * @return {number[]}
 */
var findWordsContaining = function(words, x) {
    let output = []
    for(let i = 0; i < words.length; i++) {
        if(words[i].includes(x)) {
            output.push(i)
        }
    }
    return output
};

File: 2942-find-words-containing-character/2942-find-words-containing-character.rs
impl Solution {
    pub fn find_words_containing(words: Vec<String>, x: char) -> Vec<i32> {
        let mut output = Vec::new();
        for i in 0..words.len() {
            match words[i].find(x) {
                Some(_index) => output.push(i.try_into().unwrap()),
                None => continue,
            }
        }
        output
    }
}